packages/@o3r/configuration/src/devkit/configuration-devtools.console.service.ts

```diff
@@ -20,7 +20,13 @@ export class ConfigurationDevtoolsConsoleService implements DevtoolsServiceInter
   ) {
     this.options = { ...OTTER_CONFIGURATION_DEVTOOLS_DEFAULT_OPTIONS, ...options };
 
-    if (this.options.isActivatedOnBootstrap) {
+    if (
+      this.options.isActivatedOnBootstrap
+      || (
+        this.options.isActivatedOnBootstrapWhenCMSContext
+        && (document.body.dataset as ContextualizationDataset).cmscontext === 'true'
```

Comment:
are we sure `document.body.dataset` cannot be undefined here ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset
dataset is always present on every `HTMLElement`